### ISSUE TITLE
Rule shorthand-property-no-redundant-values not working when have variables for value in sass

### ISSUE THREAD

**Issue by copiali** [kristerkari/stylelint-scss#156]
When I use variables in the value, shorthand-property-no-redundant-values rule is not working.
For example following code will not get alert:
`$baseline: 8px;
.foo {
  padding: $baseline 0 0 0;
}`

Any fix for this will be added?

**Comment by kristerkari** [comment]
That rule is part of `stylelint` and not this project. Most of the `stylelint` rules are meant to use with CSS, so they don't work with SCSS syntax. 

You might want to report this to https://github.com/stylelint/stylelint and see what they respond. It might be that it won't get fixed unless we implement a similar rule for `stylelint-scss` that supports your code example.

**Comment by copiali** [comment]
I created this issue their, but they asked me to report here...
https://github.com/stylelint/stylelint/issues/2790

**Comment by jeddy3** [comment]
> It might be that it won't get fixed unless we implement a similar rule for stylelint-scss that supports your code example.

That's correct. The built-in rules _simply ignore_ non-standard constructs. As such, a new `scss/shorthand-property-no-redundant-values` rule within this plugin pack is the way forward. Unlike the built-in rule, this new rule wouldn't ignore SCSS variables. @copiali Please consider contributing the rule to stylelint-scss if you think it'll be useful to you and others.
